examples/mqtt_client/mqtt_client.c

```diff
@@ -0,0 +1,249 @@
+
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+
+#include <string.h>
+
+#include <FreeRTOS.h>
+#include <task.h>
+#include <ssid_config.h>
+
+#include <espressif/esp_sta.h>
+#include <espressif/esp_wifi.h>
+
+#include <paho/MQTTESP8266.h>
+#include <paho/MQTTClient.h>
+
+#include <semphr.h>
+
```

Comment:
Please add a comment near the top somewhere explaining whose code this is, and what license (if any) it's under.

The convention for example code is to say they're in the public domain/CC0. You don't have to do that, though.